torch/quantization/__init__.py

```diff
@@ -18,9 +18,11 @@ def default_eval_fn(model, calib_data):
 _all__ = [
     'QuantWrapper', 'QuantStub', 'DeQuantStub',
     # Top level API for eager mode quantization
-    'quantize',
-    # Sub functions used by eager mode quantization
-    'prepare', 'convert',
+    'quantize', 'quantize_dynamic', 'quantize_qat',
+    'prepare', 'convert', 'prepare_qat',
+    # Top level API for graph mode quantization
```

Comment:
just curious, if the name is "graph mode", why not call the APIs with `_graph` instead of `_script`?

Reply:
`_script` is referring to `TorchScript`, indicating it's taking a TorchScript model. 
I think eager mode and graph mode is a separate(broader) naming convention, that is, other frameworks will have eager mode/graph mode as well, we don't really have these appear in the APIs.

Reply:
@jerryzh168 do you think changing the name to `quantize_torchscript` would be a good idea? `_script` could have a different connotation that we are scripting the model and quantizing it.

Reply:
for me "script" triggers "this is a script" instead of "this is TorchScript" as a first response.  Don't feel super strongly, but naming it more descriptively might be valuable.

Reply:
that is too long I think. I think people still need to read the doc to know how to use it anyways, the name shouldn't be that important.

Reply:
what about `quantize_jit` or `quantize_ts`?

Reply:
> quantize_jit

I really like that one, because JIT in the context of PyTorch is very specific

Reply:
sounds good, will put up a rename PR